x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -4,9 +4,9 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { CURRENT_MAJOR_VERSION } from '../../../common/version';
+import { docLinksService } from '../shared/doc_links/doc_links';
```

Comment:
nit: just import directly from the folder index.ts

```suggestion
import { docLinksService } from '../shared/doc_links'; 
```